front/app/components/UI/UserName/index.tsx

```diff
@@ -26,89 +64,86 @@ interface StyleProps {
   canModerate?: boolean;
 }
 
-interface InputProps extends StyleProps {
+interface Props extends StyleProps {
```

Comment:
I know this is how we do it elsewhere, but what do you think about starting to use more explicit names for the Props - in this case `UserNameProps`? It makes it easier to search for and if, for some reason you need to export the type, there is no need to rename it at the import location.

Reply:
Personally, I haven't needed this in +3 years, but I don't have an argument against. 🙂 Except perhaps one: if you decide to rename a component, you also have to rename the props everywhere.